src/libcore/hash/sip.rs

```diff
@@ -193,6 +193,14 @@ impl Hasher for SipHasher {
     }
 
     #[inline]
+    fn delimit(&mut self, len: usize) {
+        // skip the first delimiter
+        if self.length > 0 {
+            self.write_usize(len);
```

Comment:
I think it's a good example, it seems to be a flooding attack. It's properly delimited by One on its own.